WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -824,7 +829,8 @@ templates.class.HUNTER = {
         { spell = 199483, type = "buff", unit = "player", talent = 9}, -- Camouflage
         { spell = 24450, type = "buff", unit = "pet"}, -- Prowl
         { spell = 136, type = "buff", unit = "pet"}, -- Mend Pet
-
```

Comment:
same here

Reply:
I removed the empty line in this place, should I add it again?

Reply:
It's fine, Stan just misinterpreted the diff.